lms/migrations/versions/7e4124035651_add_the_provisioning_column.py

```diff
@@ -0,0 +1,31 @@
+"""
+Add the provisioning column.
+
+Revision ID: 7e4124035651
+Revises: efcf8671f4d3
+Create Date: 2018-12-07 11:38:55.717986
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "7e4124035651"
+down_revision = "efcf8671f4d3"
+
+
+def upgrade():
+    op.add_column(
+        "application_instances",
+        sa.Column(
+            "provisioning",
+            sa.Boolean(),
+            server_default=sa.sql.expression.true(),
```

Comment:
Oooh, I wish there were, like, an SQLAlchemy/alembic cookbook somewhere because this is one of those things that's obvious when you read it but might take a dev like me a while to dig out of docs.